sources/packages/api/src/route-controllers/student/models/student.dto.ts

```diff
@@ -0,0 +1,22 @@
+
+import { IsNotEmpty, IsOptional} from 'class-validator';
+
+export class CreateStudentDto {
+  @IsNotEmpty()
+  phone: string;
+  @IsNotEmpty()
+  sinNumber: string;
```

Comment:
We need to add server validation for the SIN Number also. Can you create a tech debt for it?